remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/RemoteBuildCoordinator.java

```diff
@@ -146,41 +163,43 @@ public RemoteBuildCoordinator(
      * @throws BuildConflictException If there is already a build running with the same build configuration Id and
      *         version
      */
+    @Transactional
+    @Retry(retryOn = ScheduleConflictException.class)
+    @Fallback(BuildConfigurationAuditedFallback.class)
```

Comment:
With Fallback, practically every Exception is wrapped as RuntimeException because it catches every Exception from under its methods... Is this too much of a problem?

Reply:
We can add skipOn exceptions if we do not want to invoke the fallback on some Exceptions and applyOn to specify specific exception we do want fallback to apply on. I think it would have been cool if we had ALL exception handling in Fallback but due to inability to throw checked exceptions it is not feasible :/ .

Reply:
Removed retry, there is no need to handle to store the result when the retries are exhausted (schedule config).